openrtb.go

```diff
@@ -690,6 +690,13 @@ const (
 	DeviceTypeSetTopBox DeviceType = 7
 )
 
+var deviceTypes = [...]string{"0", "1", "2", "3", "4", "5", "6", "7", "8", "9"}
+
+func (dt DeviceType) String() string {
```

Comment:
1. so I am not sure that String is the right method, maybe `Code() string`. 
2. you should also check `if dt >= DeviceTypeUnknown && dt <= DeviceTypeSetTopBox` to make sure you don't get an accidental panic. `return DeviceTypeUnknown` otherwise
3. why is deviceTypes going up to 9, there are only 7 of them?
4. you could define `cost deviceTypeCodes = "01234567"` for now and use string indexing which is a little more efficient